amato/umbrel-app.yml

```diff
@@ -0,0 +1,21 @@
+manifestVersion: 1
+id: amato
```

Comment:
pomodoro or amato?

Reply:
[Pomodoro](https://en.wikipedia.org/wiki/Pomodoro_Technique) is the technique of dedicated focus. Often (for reasons that I don't fully understand), "Pomodoro time" is often signified by a tomato. For example, if you are doing focused work and do not want to be bothered, you might set your Slack status to a tomato emoji.

My guess is "Amato" is part of the brand that this developer wants to have for his pomodoro timer, a reference to the tomato. If he named his app Pomodoro, it would be a lot more competitive from an SEO perspective.